src/main/ssp/codegen/scala/class_companion.ssp

```diff
@@ -60,6 +60,15 @@ class ${cls.name}Meta
 #end
     ret
   }
+<% val indexes = cls.annotations.toSeq.filter(_._1 == "index").map(_._2) %>
+#if (indexes.nonEmpty)
+  object indexes {
+#for (index <- indexes)
+    lazy val `${index.replace(": asc", "Asc").replace(": desc", "Desc").replace(",", "").replace(" ", "")}` =
```

Comment:
there's no way this is general enough. we definitely have indexes that look like "user:asc,campaign:asc,state:asc"